deps.sh

```diff
@@ -2,6 +2,9 @@
 # Add new direct and transitive dependencies as submodules.
 # Note: caller must validate/curate/commit changes to .gitmodules and 'vendor/'
 #
+echo "Note: caller must validate/curate/commit changes to .gitmodules and 'vendor/'"
```

Comment:
Is there a way to run make test in a way where it *won't* get/install what's needed so we'll get a loud break in TravisCI instead of "oh it silently works but without versioning"?